views/sms-settings.ejs

```diff
@@ -90,6 +90,32 @@
         <select id="bday-year" name="bday-year"></select>
       </div>
     </fieldset>
+    
+    <fieldset>
+      <div class="label-container">
+        <label>What's one thing you want to work on most?</label>
```

Comment:
The copy for the labels and menu options pretty much come from the copy currently being used for intention-setting flow over SMS.